packages/cactus-common/src/main/typescript/logging/logger.ts

```diff
@@ -46,6 +118,24 @@ export class Logger {
     this.backend.info("Shut down logger OK.");
   }
 
+  /**
```

Comment:
@m-courtin Please clarify in the docs the value add of this method compared to `logger.error(ex);` (e.g. one is human readable [1] the other is machine readable)

I'd also make sure to let people know that it's a wrapper around `logger.error` so that they don't have to wonder what level it will get logged at or if it gets logged at all since `exception` (as-in the method name itself) is a noun not a verb. 

[1]

```sh
console.error
    [2022-02-07T23:02:51.905Z] ERROR (TestX): r [RuntimeError]: Doh
        at Function.newRuntimeError (/home/peter/a/blockchain/cactus/packages/cactus-common/src/main/typescript/exception/exception-helper.ts:45:22)
        at /home/peter/a/blockchain/cactus/packages/cactus-common/src/test/typescript/unit/exception/exception-helper.test.ts:340:32
        at _dispatchDescribe (/home/peter/a/blockchain/cactus/node_modules/jest-circus/build/index.js:97:26)
        at describe (/home/peter/a/blockchain/cactus/node_modules/jest-circus/build/index.js:60:5)
        at /home/peter/a/blockchain/cactus/packages/cactus-common/src/test/typescript/unit/exception/exception-helper.test.ts:336:3
        at _dispatchDescribe (/home/peter/a/blockchain/cactus/node_modules/jest-circus/build/index.js:97:26)
        at describe (/home/peter/a/blockchain/cactus/node_modules/jest-circus/build/index.js:60:5)
        at Object.<anonymous> (/home/peter/a/blockchain/cactus/packages/cactus-common/src/test/typescript/unit/exception/exception-helper.test.ts:7:1)
        at Runtime._execModule (/home/peter/a/blockchain/cactus/node_modules/jest-runtime/build/index.js:1646:24)
        at Runtime._loadModule (/home/peter/a/blockchain/cactus/node_modules/jest-runtime/build/index.js:1185:12) {
      cause: r [RuntimeError]: Oops
          at /home/peter/a/blockchain/cactus/packages/cactus-common/src/test/typescript/unit/exception/exception-helper.test.ts:341:7
          at _dispatchDescribe (/home/peter/a/blockchain/cactus/node_modules/jest-circus/build/index.js:97:26)
          at describe (/home/peter/a/blockchain/cactus/node_modules/jest-circus/build/index.js:60:5)
          at /home/peter/a/blockchain/cactus/packages/cactus-common/src/test/typescript/unit/exception/exception-helper.test.ts:336:3
          at _dispatchDescribe (/home/peter/a/blockchain/cactus/node_modules/jest-circus/build/index.js:97:26)
          at describe (/home/peter/a/blockchain/cactus/node_modules/jest-circus/build/index.js:60:5)
          at Object.<anonymous> (/home/peter/a/blockchain/cactus/packages/cactus-common/src/test/typescript/unit/exception/exception-helper.test.ts:7:1)
          at Runtime._execModule (/home/peter/a/blockchain/cactus/node_modules/jest-runtime/build/index.js:1646:24)
          at Runtime._loadModule (/home/peter/a/blockchain/cactus/node_modules/jest-runtime/build/index.js:1185:12)
          at Runtime.requireModule (/home/peter/a/blockchain/cactus/node_modules/jest-runtime/build/index.js:1009:12) {
        cause: 'Snap!'
      }
    }
```

Reply:
@petermetz: I added some documentation to describe that the exception function is retrieving exception content in safe and secure way whatever exception is thrown and that there is no further need to check on stack trace, dealing with different exception types etc in code.